modules/hcp-support-matrix.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-prepare/hcp-requirements.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-support-matrix"_{context}"]
+= Support matrix for {hcp}
+
+Because {mce} includes the HyperShift Operator, releases of {hcp} align with releases of {mce-short}. For more information, see "OpenShift Operator Life Cycles".
+
+Any supported {product-title} cluster can be a management cluster. The following table maps {mce-short} versions to the management cluster versions that support them:
+
+.Supported {mce-short} versions for {product-title} management clusters
+[cols="2",options="header"]
+|===
+|Management cluster version |Supported {mce-short} version
+
+|4.12 - 4.15
+|2.4
+
+|4.13 - 4.16
+|2.5
+
+|4.14 - 4.17
+|2.6
+
+|4.15 - 4.17
+|2.7
+|===
+
+For hosted clusters, no direct relationship exists the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator of your {mce-short} version. The following table maps {mce-short} versions to which hosted cluster versions you can create by using the HyperShift Operator that is associated with that version of {mce-short}:
+
+.Hosted cluster versions that can be created by {mce-short} versions
+[cols="5",options="header"]
+|===
+|Hosted cluster version  |{mce-short} 2.4 |{mce-short} 2.5 |{mce-short} 2.6 |{mce-short} 2.7
+
+|4.14
+|Yes
+|Yes
+|Yes
+|Yes
+
+|4.15
+|No
+|Yes
+|Yes
+|Yes
+
+|4.16
+|No
+|No
+|Yes
+|Yes
+
+|4.17
+|No
+|No
+|No
+|Yes
+|===
+
+If you have a running hosted cluster and you update {mce-short} to a later version, provided that the HyperShift Operator supports the hosted cluster version, the hosted cluster is not affected. The following table shows the affects of {mce-short} updates on various versions of hosted clusters:
+
+.Affects of {mce-short} updates on hosted clusters
+[cols="4",options="header"]
+|===
+|Hosted cluster version| Update from {mce-short} 2.4 to 2.5 |Update from {mce-short} 2.5 to 2.6 |Update from {mce-short} 2.6 to 2.7
+
+|4.14
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+
+|4.15
+|Hosted cluster is affected
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+
+|4.16
+|Hosted cluster is affected
+|Hosted cluster is affected
+|No affect, hosted cluster continues to run
+
+|4.17
+|Hosted cluster is affected
+|Hosted cluster is affected
+|Hosted cluster is affected
+|===
```

Comment:
The idea to communicate here is that as long as the HO shipped with the MCE version being upgraded to can support the version of the HostedCluster, the cluster can still be lifecycled (nothing would happen). 

So I recommend removing "Hosted cluster is affected" 

For example, upgrading from 2.4 to 2.5 while having a 4.14 hosted cluster on the management cluster will not impact the existing 4.14 cluster. Why? because the Hypershift operator shipped with MCE 2.5 can lifecycle both 4.15 AND 4.14 clusters. 

Hope that helps.